lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb

```diff
@@ -1,8 +1,9 @@
 <% unless sidebar.archives.blank? %>
-  <h3 class="sidebar_title"><%= sidebar.title %></h3>
-  <div class="sidebar_body">
-    <ul id="archives">
```

Comment:
Consistency with the other `<ul>`s on the page that don't have ids. 

![screen shot 2016-02-05 at 11 21 58 am](https://cloud.githubusercontent.com/assets/15789101/12856631/b4e7034e-cbfa-11e5-9d7f-56602bac709a.png)
![screen shot 2016-02-05 at 11 21 43 am](https://cloud.githubusercontent.com/assets/15789101/12856642/be830d26-cbfa-11e5-929e-b9d74f5d01b0.png)